### PR TITLE
Subaru: add infotainment status message

### PR DESCRIPTION
**Pull request by jnewb1**
d9df6f87e8feff94|2023-03-28--17-41-10--0

this route has the transitions marked using user flags

goal is to be able to filter out the stock message, and also display our own

image
IMG_0145

### CODE REVIEW

File: generator/subaru/subaru_global_2017.dbc [hunk 1]
```diff
@@ -53,10 +53,19 @@ BO_ 576 CruiseControl: 8 XXX
...
+ SG_ SIGNAL_3 : 23|1@0+ (1,0) [0|1] "" XXX
+
 CM_ SG_ 545 Cruise_Throttle "RPM-like output signal";
 CM_ SG_ 545 Cruise_EPB "1 = Electric Parking Brake set";
 CM_ SG_ 545 Distance_Swap "Switch from Close to Far distance";
 CM_ SG_ 545 Cruise_Soft_Disable "Eyesight Temporary disable, sets CruiseControl Activated = 0";
 CM_ SG_ 546 Cruise_RPM "ES RPM output for ECM and TCM";
 CM_ SG_ 546 Signal3 "0 when cruise_activated = 1";
-VAL_ 72 Gear 2 "N" 3 "R" 4 "P" 121 "D" 137 "1" 145 "2" 153 "3" 161 "4" 169 "5" 177 "6";
+CM_ SG_ 803 LKAS_State_Infotainment "0 = none; 2 = Obstacle Detected; 3 = Keep Hands On Wheel; 4 = Keep Hands On Wheel Off";
```
Comment (adeebshihadeh) [inline generator/subaru/subaru_global_2017.dbc:70]: can you convert this comment into VAL_ values?

File: generator/subaru/subaru_global_2017.dbc [hunk 2]
```diff
@@ -53,10 +53,19 @@ BO_ 576 CruiseControl: 8 XXX
  SG_ Cruise_Activated : 41|1@1+ (1,0) [0|1] "" XXX
  SG_ Signal2 : 42|22@1+ (1,0) [0|4194303] "" XXX
 
+BO_ 803 INFOTAINMENT_STATUS: 8 XXX
+ SG_ CHECKSUM : 0|8@1+ (1,0) [0|255] "" XXX
+ SG_ COUNTER : 8|4@1+ (1,0) [0|15] "" XXX
+ SG_ SIGNAL_1 : 15|4@0+ (1,0) [0|15] "" XXX
+ SG_ SIGNAL_2 : 19|44@0+ (1,0) [0|3.51844e+13] "" XXX
```
Comment (adeebshihadeh) [inline generator/subaru/subaru_global_2017.dbc:60]: are these signals meaningful? if not, let's garbage collect them

**Review by adeebshihadeh — CHANGES_REQUESTED** [review]

File: generator/subaru/subaru_global_2017.dbc (hunk 2 above)
Comment (jnewb1) [reply generator/subaru/subaru_global_2017.dbc]: done

File: generator/subaru/subaru_global_2017.dbc (hunk 1 above)
Comment (jnewb1) [reply generator/subaru/subaru_global_2017.dbc]: done

**Pull request merged**